modules/project-factory/variables.tf

```diff
@@ -408,3 +442,15 @@ variable "factories_data" {
   nullable = false
   default  = {}
 }
+
+variable "federated_identity_pool" {
```

Comment:
Here we should conform to the format we use in FAST, we have expose a map of federated identies. We don't need two variables one for the pool one for the provider. But I can see situations where a project factory manages projects that use different providers or pools.

Reply:
This is now passed down via context and we need the pool name to construct the `sa_wif_binding`.